src/Magento/FunctionalTestingFramework/Console/RunManifestCommand.php

```diff
@@ -108,15 +108,20 @@ protected function execute(InputInterface $input, OutputInterface $output): int
      */
     private function runManifestLine(string $manifestLine, OutputInterface $output)
     {
-        $codeceptionCommand = realpath(PROJECT_ROOT . "/vendor/bin/codecept")
-            . " run functional --verbose --steps "
-            . $manifestLine;
+        $codeceptionCommand = realpath(PROJECT_ROOT . "/vendor/bin/codecept") . " run functional --verbose --steps ";
+        if (getenv('ENABLE_PAUSE') === 'true') {
+            $codeceptionCommand .= ' --debug';
+        }
+        $codeceptionCommand .= $manifestLine;
 
         // run the codecept command in a sub process
         $process = new Process($codeceptionCommand);
         $process->setWorkingDirectory(TESTS_BP);
         $process->setIdleTimeout(600);
         $process->setTimeout(0);
+        if (getenv('ENABLE_PAUSE') === 'true') {
```

Comment:
In the interactive console, are you able to use the function keys and TAB keys. I remember I was able to use them with codeception --debug.
ENTER to continue
  - TAB to auto-complete
  - F5 to stash a command
  - F6 to toggle auto-stashing of successful commands
  - F8 to view stashed commands
  - F10 to clear stashed commands

Reply:
We don't have control over the key mappings. But I think I might know the reason for the Terminal hanging. Good catch!

Reply:
Do you mean we won't be able to use function and tab keys in the interactive console?

Reply:
I thought I can do something for the Terminal hanging. But it looks like it's coming from Codeception. Same with the key mappings. Both issue can be reproducible if you run Codeception through PhPStorm.